### PR TITLE
Initial frontend cms

### PR DESCRIPTION
**Pull request by jacknugent**
image

Done:
- Display bills/issues
- Update bills/issues
- Create bills
- Creating issues
- Deleting issues/bills
- Adding talking points to issues

Next up:
- Connect to deployment (initial setup just like CCA frontend)
- Get feedback for table/modal content/order of properties
- Get feedback for layout

### CODE REVIEW

File: frontend_cms/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```
Comment (kevincianfarini) [inline frontend_cms/README.md:1]: Can we delete this file?

File: frontend_cms/public/robots.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```
Comment (kevincianfarini) [inline frontend_cms/public/robots.txt:3]: I don't think we want to allow scraping here. Can we disallow everything?

File: frontend_cms/src/Components/CongressionalBills/CongressionalBillsModal.tsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+const BILL_TYPES = [
+    "H.R.",
+    "H.Con.Res.",
+    "H.J.Res.",
+    "H.Res.",
+    "S.",
+    "S.Con.Res.",
+    "S.J.Res.",
+    "S.Res.",
+];
```
Comment (kevincianfarini) [inline frontend_cms/src/Components/CongressionalBills/CongressionalBillsModal.tsx:17]: These should be fetched from the Bill API. If we don't have one yet, I should add it.

File: frontend_cms/README.md (hunk 1 above)
Comment (jacknugent) [reply frontend_cms/README.md:1]: Hmm we could but I also think it's kind of helpful. Would be better to have CCA specific README but I think this generated one is better than nothing

Comment (kevincianfarini) [reply frontend_cms/README.md:1]: CCA specific would be good!

File: frontend_cms/src/Components/CongressionalBills/CongressionalBillsModal.tsx (hunk 3 above)
Comment (jacknugent) [reply frontend_cms/src/Components/CongressionalBills/CongressionalBillsModal.tsx:17]: Unless I'm mistaken there isn't one yet

File: frontend_cms/public/robots.txt (hunk 2 above)
Comment (jacknugent) [reply frontend_cms/public/robots.txt]: Added `/` to robots.txt which disallows scraping

File: frontend_cms/README.md (hunk 1 above)
Comment (jacknugent) [reply frontend_cms/README.md]: Done

File: frontend_cms/src/Components/CongressionalBills/CongressionalBillsModal.tsx (hunk 3 above)
Comment (kevincianfarini) [reply frontend_cms/src/Components/CongressionalBills/CongressionalBillsModal.tsx:17]: I will add it.

Comment (kevincianfarini) [reply frontend_cms/src/Components/CongressionalBills/CongressionalBillsModal.tsx:17]: Resolved. #387

**Comment by kevincianfarini** [comment]
This looks like a great first go. I have a few small nitpicks, but they can be addressed a bit later. 

1. The display state of a bill should be the propert bill name/function. Eg. "H.R. 2307 The Energy Innovation and Carbon Dividend Act of 2021". Right now I think it's using the database ID and looks like "1 - Energy Innovation and Carbon Dividend Act of 2021"
2. It would be great if the update modal for both bills and issue would be wider. This is especially important for drafting talking points within issues. 

I think those are my only comments for now!

**Review by kevincianfarini — APPROVED** [review]

**Pull request merged**